spec/features/admin/business_units_spec.rb

```diff
@@ -12,12 +12,12 @@
     teams_index_page.load
     business_group_row =
       teams_index_page.row_for_business_group bu.business_group.name
-    expect(business_group_row.name.text).to eq(bu.business_group.name)
+    expect(business_group_row.name.text).to have_text "#{bu.business_group.name}"
```

Comment:
And this was the other thing - trying this just as 
```suggestion
    expect(business_group_row.name.text).to have_text bu.business_group.name
```